codegen-core/common-test-models/malformed-range-extras.smithy

```diff
@@ -0,0 +1,662 @@
+$version: "2.0"
+
+namespace aws.protocoltests.extras.restjson.validation
+
+use aws.api#service
+use aws.protocols#restJson1
+use smithy.test#httpMalformedRequestTests
+use smithy.framework#ValidationException
+
+/// A REST JSON service that sends JSON requests and responses with validation applied
+@service(sdkId: "Rest Json Validation Protocol")
+@restJson1
+service MalformedRangeValidation {
+    version: "2022-11-23",
+    operations: [
+        MalformedRange,
+        MalformedRangeOverride,
+    ]
+}
+
+@suppress(["UnstableTrait"])
```

Comment:
I've mimicked the annotations on https://github.com/awslabs/smithy/blob/a8fd6c537d3d1e0c0258741efceda3ea85a8c75b/smithy-aws-protocol-tests/model/restJson1/validation/malformed-range.smithy#L16 since we plan to merge this upstream later on. I'm not sure what the purpose of it is in their own model.